### ISSUE TITLE
Steam won't start on Ubuntu 14.04

### ISSUE THREAD

**Issue by ProhibitedUser** [ValveSoftware/steam-for-linux#3309]
It worked with 13.10, but not anymore. I've tried uninstalling and reinstalling but it changes nothing. When I try to start Steam from the Terminal I get this:
thomas@thomas-desktop:~$ steam
Running Steam on ubuntu 14.04 64-bit
STEAM_RUNTIME is enabled automatically
Installing breakpad exception handler for appid(steam)/version(0_client)
Uploading dump (out-of-process) [proxy '']
/tmp/dumps/crash_20140517164638_1.dmp
/home/thomas/.local/share/Steam/steam.sh: line 755:  5420 Segmentation fault      (core dumped) $STEAM_DEBUGGER "$STEAMROOT/$PLATFORM/$STEAMEXE" "$@"
mv: cannot stat ‘/home/thomas/.steam/registry.vdf’: No such file or directory
Installing bootstrap /home/thomas/.local/share/Steam/bootstrap.tar.xz
Reset complete!
Restarting Steam by request...
Running Steam on ubuntu 14.04 64-bit
STEAM_RUNTIME has been set by the user to: /home/thomas/.local/share/Steam/ubuntu12_32/steam-runtime
Installing breakpad exception handler for appid(steam)/version(0_client)
Uploading dump (out-of-process) [proxy '']
/tmp/dumps/crash_20140517164640_1.dmp
/home/thomas/.local/share/Steam/steam.sh: line 755:  5547 Segmentation fault      (core dumped) $STEAM_DEBUGGER "$STEAMROOT/$PLATFORM/$STEAMEXE" "$@"
Finished uploading minidump (out-of-process): success = yes
response: CrashID=bp-7b04ba96-1ac5-432a-a9cc-350312140517
Finished uploading minidump (out-of-process): success = no
error: Couldn't connect to server

There's some more info here: https://support.steampowered.com/view.php?ticketref=1207-ETYO-8842

In short they referred me to this site. This problem has been bugging me for a month now and I would really appreciate any ideas anyone has about fixing it.

**Comment by kisak-valve** [comment]
Closing as general video driver quality from 2014 is no longer relevant.
